package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "happo.io",
-  "version": "6.8.0",
+  "version": "6.9.0",
```

Comment:
in open source projects, versions are generally not bumped in PRs, rather only at release time

Reply:
Agreed. I like how you're thinking about the semver implications of this feature, but I think a comment in the PR description would have been enough. 

You can remove this line. I'm working on some other changes as well right now and plan to release 7.0.0 soon (might happen tomorrow or after the holidays). I'll include your changes in that release.